packages/snap-preact/src/Templates/Stores/LibraryStore.ts

```diff
@@ -80,20 +82,25 @@ export class LibraryStore {
 					);
 				},
 			},
-			recommendation: {
-				Recommendation: async () => {
+			'recommendation.bundle': {
+				RecommendationBundle: async () => {
 					return (
-						this.components.recommendation.Recommendation ||
-						(this.components.recommendation.Recommendation = (await import('./library/components/Recommendation')).Recommendation)
+						this.components['recommendation.bundle'].RecommendationBundle ||
+						(this.components['recommendation.bundle'].RecommendationBundle = (
+							await import('./library/components/RecommendationBundle')
+						).RecommendationBundle)
 					);
 				},
-				RecommendationBundle: async () => {
+			},
+			'recommendation.default': {
+				Recommendation: async () => {
 					return (
-						this.components.recommendation.RecommendationBundle ||
-						(this.components.recommendation.RecommendationBundle = (await import('./library/components/RecommendationBundle')).RecommendationBundle)
+						this.components['recommendation.default'].Recommendation ||
+						(this.components['recommendation.default'].Recommendation = (await import('./library/components/Recommendation')).Recommendation)
```

Comment:
Is there any reason to not actually go another level deeper in the object instead of using a `.` to delimit the path superficially?

Reply:
No particular reason. I think I'd actually prefer a `/` instead of `.` since we don't do this else where. Nested object would also work

Reply:
Unless there is good reason, I think my preference here is nested object.